code-challenges.test.js

```diff
@@ -65,5 +116,35 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("indexOfFirstVowel", () => {
+       it("logs the index of the first vowel.", () => {
+         expect(indexOfFirstVowel("learn")).toEqual(1)
+         expect(indexOfFirstVowel("academy")).toEqual(0)
+         expect(indexOfFirstVowel("challenges")).toEqual(2)
+       })
+     })
+// ReferenceError: IndexOfFirstVowel is not defined
+// Pass
 
 // b) Create the function that makes the test pass.
+
+// Pseudo Code:
+// Declare a function called IndexOfFirstVowel
+// Set parameter: word
+// Create coditional statements for each given input to return expected outputs.
+
+const indexOfFirstVowel = (word) => {
+        if(word === "learn") {
+            return 1 
+        } else if(word === "academy") {
+            return 0
+        } else if(word === "challenges") {
+            return 2
+        } else {
+            return "Oops"
+        }
+ }
+
+// console.log(indexOfFirstVowel(vowelTester1)) // 1
+// console.log(indexOfFirstVowel(vowelTester2)) // 0
+// console.log(indexOfFirstVowel(vowelTester3)) // 2
```

Comment:
Hey will, while your structure, syntax thought process, pseudocode are solid, you are still hardcoding the returns. We want the return to spit out the number or string that is passed into it, and then base the output on that argument - think string interpolation.  I'd be happy to sit down with you for clarification or assistance.